### DISCUSSION TITLE
ComfyUI-Manager window is not visible

### DISCUSSION THREAD

**Discussion by ZonSoft** [Comfy-Org/ComfyUI-Manager#44] [Q&A]
I made Manager window to disappear somehow. Manager loads correctly but window is not there. Reinstall did not help. It can be offscreen somewhere but I have no idea how to reset coordinates - or it may be some other reason... No useful info in the log, everything looks OK - just no Manager window.
Any ideas?

**Comment by ZonSoft** [comment]
Correction. There is no window with "Queue prompt" button - if it's correct to call it Manager window.

**Reply by ltdrdata** [reply]
Do you meaning 'Main menu' is missing?

**Reply by ZonSoft** [reply]
Yes. Reinstalling does not help.

**Reply by ZonSoft** [reply]
Q & H do not help either - just in case. Maybe my question is not posted to the right place, but any ideas are appreciated.

**Reply by ZonSoft** [reply]
Managed to resolve, sorry for bothering. Clearing cookies for 127.0.0.1 in browser fixed the issue.
